### PR TITLE
Fixed activateROS for catking build and conda envs. Updated readme with uninstall

### PR DESCRIPTION
**Pull request by vonHartz**
See title. See also #7 . Catkin build achieved compatibility by removing the check for CMakeLists.txt. Is there some other test you would want to add @ARoefer (such as src, devel dirs) or is checking for the venv enough?

### CODE REVIEW

**Comment by ARoefer** [comment]
Thank you for addressing this! I have so far not been able to come up with another reliable check. Maybe the combination of `src` and `pyenv`? But maybe the check for `pyenv` is enough, as it seems to be an atypical name for venvs. Alternative solution would be that we put an empty file at the root of a workspace. Something like `.rosvenv_ws`. What do you think?

File: README.md [hunk 1]
```diff
@@ -33,6 +33,14 @@ You can also do `./install_rosvenv.bash`, but then you'll have to re-source `.ba
 
 What did the install script just do to your system you ask? Well, it simply copied the file `rosvenv.bash` to `~/.rosvenv.bash` and added `source ~/.rosvenv.bash` to your `~/.bashrc`. Re-running the script will only perform the copy again, but will not modify your `.bashrc` unless the function `createROSWS` is nowhere to be found.
 
+# Uninstall
```
Comment (ARoefer) [inline README.md:36]: Thank you for this :)

**Comment by vonHartz** [comment]
> Alternative solution would be that we put an empty file at the root of a workspace. Something like `.rosvenv_ws`. What do you think?

This feels like unnecessary complexity to me, so I'd rather avoid it.

> Maybe the combination of src and pyenv?

Yeah, in my workspace I only have `build  condenv.txt  devel  logs  src`. But folders like `src` are so generic that there's probably little value in checking for them. So checking for pyvenv/condaenv.txt is likely enough.

Also, there's little damage done by not checking well enough at this stage. Worst case is that something like ws sourcing will fail. So maybe, we can extend the error message that is generated in such a case to point to the possibility that the ws dir is faulty.

**Comment by ARoefer** [comment]
I think you're right, there's such a thing as overthinking it. I'll merge it now and then we can improve in the future. (I wan't to improve the docker tooling anyway some time in July)

**Pull request merged**